examples/example-app-router/src/i18n/request.ts

```diff
@@ -1,6 +1,29 @@
 import {notFound} from 'next/navigation';
 import {getRequestConfig} from 'next-intl/server';
 import {routing} from './routing';
+import { Formats } from 'next-intl';
```

Comment:
Can you execute [the linter](https://github.com/amannn/next-intl/blob/main/CONTRIBUTORS.md#code-formatting-eslint--prettier) after your changes?

- [ ] Fix code formatting

Reply:
Yeah, so I am having some problems with formatting the code. I had setup the defaultFormatter/formatOnSave on user `settings.json` based on the CONTRIBUTORS.md file, but still it didn't work. I managed to format it by using the cli command though.

![image](https://github.com/user-attachments/assets/a102c159-59fa-4b1e-8686-1b15e193e6ec)

Why use eslint for formatting instead of prettier?

Reply:
The common issue that people run into is this:

> For ESLint to work correctly, you should open individual packages you work on as the workspace root (e.g. `./packages/next-intl`).

But yes, I know the toolchain is a bit controversial. Just a note that Prettier is still used, however via ESLint.

I'd like to get rid of this though and have opened https://github.com/amannn/next-intl/issues/1349 now. Thanks for the nudge!

Reply:
Ok, I more or less managed to make it work. Wasn't familiar with this type of workflow before. I am having to use >format Document (forced) for now